content/ADR-253-light-sources.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: adr
+adr: 253
+title: Light Sources
+date: 2024-11-13
+status: Review
```

Comment:
Probably at this stage this should be a draft

Reply:
Addressed on last commit